gfx/drivers/gl.c

```diff
@@ -864,10 +862,8 @@ static GLenum gl2_min_filter_to_mag(GLenum type)
       case GL_NEAREST_MIPMAP_NEAREST:
          return GL_NEAREST;
       default:
-         break;
```

Comment:
I'd rather you change this back to what it was.

I'd rather have a clear return at the end of the function instead of leaving it empty like that.